crates/api_crud/src/comment/update.rs

```diff
@@ -95,15 +90,14 @@ impl PerformCrud for EditComment {
     )
     .await?;
 
-    send_comment_ws_message(
-      data.comment_id,
-      UserOperationCrud::EditComment,
-      websocket_id,
-      data.form_id.to_owned(),
-      None,
+    let comment_view = blocking(context.pool(), move |conn| {
+      CommentView::read(conn, comment_id, Some(person_id))
+    })
+    .await??;
+    Ok(CommentResponse {
+      comment_view,
       recipient_ids,
-      context,
-    )
-    .await
```

Comment:
We don't want to stop it from sending websockets.... really we want the `send_local_notifs` on line 72 to not send anything. But that also builds the recipient ids, which websocket requires. There's a `do_send_email` bool on that function, just set it to false I think, and maybe add a comment.

Reply:
This file still needs that do_send_email to be set to false. Other than that, looks good.

Reply:
You are commenting on an outdated diff, the PR now doesnt change the file at all. And on main it already has do_send_email false.

Reply:
Ah, cool.